docs/userGuide/syntax/panels.mbdf

```diff
@@ -137,7 +137,8 @@
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">html</variable>
 <variable name="code">
-<panel header="Content loaded in from 'src'" src="extra/loadContent.html#fragment" minimized></panel>
```

Comment:
Current behaviour (after #1548): 
- `<panel minimized></panel>` has hydration issues
- `<panel minimized>\n</panel>` works fine
- Normal panels without minimized work fine

Reply:
👀 hmm that seems odd. What's causing the difference between 1 & 2?

Reply:
Hi @ang-zeyu! As mentioned, I think it has something to do with getting a block element into `<p>` tag, which will cause hydration issues. 

Since in case 1, there's no line break to "end" the starting `<p>` tag, then it is likely that some block level element from panel component may get caught in it. 

In case 2, since there's a link break at the start, then the block level components won't get caught into the `<p>` tag, since it has already closed